docs/reference/logging/index.md

```diff
@@ -25,6 +25,16 @@ container's logging driver. The following options are supported:
 
 The `docker logs`command is available only for the `json-file` logging driver.  
 
+### The json-file options
+
+The following logging options are supported for the `json-file` logging driver:
```

Comment:
Can you change this line to the example that @moxiegirl gave in https://github.com/docker/docker/pull/11485#discussion_r34721477 ?